scaleway/resource_rdb_acl.go

```diff
@@ -218,6 +222,31 @@ func rdbACLExpand(data []interface{}) ([]*rdb.ACLRuleRequest, error) {
 	return res, nil
 }
 
+func rdbACLRulesFlattenFromSchema(rules []*rdb.ACLRule, dataFromSchema []interface{}) []map[string]interface{} {
```

Comment:
This looks a lot like a merge operation. Should it be renamed to convey this idea?

Reply:
Good to me